.travis.yml

```diff
@@ -7,7 +7,7 @@ osx_image: xcode8
 
 install:
     - npm install
-    - npm install appium@1.5.3
+    - npm install appium@1.7.2
```

Comment:
I guess this does not work because this repo is not supporting the version yet.

Reply:
Is it ok if I will make it work with latest?

Reply:
Yes, it's so helpful to make it work.